docroot/themes/contrib/civictheme/civictheme.provision.inc

```diff
@@ -100,17 +102,9 @@ function _civictheme_form_system_theme_settings_submit_provision(array &$form, F
     if (!empty($types)) {
       $results = civictheme_provision($types, $clear_cache);
       foreach ($results as $type => $result) {
-        if ($result === TRUE) {
-          \Drupal::messenger()->addStatus(t('Successfully provisioned @type.', [
-            '@type' => $type,
-          ]));
-        }
-        else {
-          \Drupal::messenger()->addError(t('Unable to provision @type: @message.', [
-            '@type' => $type,
-            '@message' => $result ?: t('Unknown reason'),
-          ]));
-        }
+        $result
```

Comment:
please add back strict type comparison `=== TRUE`

See the function descripotion

Reply:
Good catch, I added the extra check now